airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -975,6 +975,22 @@ public Optional<ActorCatalogFetchEvent> getMostRecentActorCatalogFetchEventForSo
     return records.stream().findFirst().map(DbConverter::buildActorCatalogFetchEvent);
   }
 
+  public Map<UUID, ActorCatalogFetchEvent> getMostRecentActorCatalogFetchEventForSources(final List<UUID> sourceIds)
```

Comment:
@benmoriceau what is this method supposed to do? the argument that is passed in, is unused.

Reply:
@cgardens. It is doing a similar operation than `getMostRecentActorCatalogFetchEventForSource` but for a list of sourceIds instead of a single source Id. The use of the input got lost during PR updates

Fixed in https://github.com/airbytehq/airbyte/pull/19668